library/src/main/java/com/novoda/downloadmanager/PartialDownloadMigrationExtractor.java

```diff
@@ -14,12 +14,10 @@
     private static final int TITLE_COLUMN = 1;
     private static final int MODIFIED_TIMESTAMP_COLUMN = 2;
 
-    private static final String DOWNLOADS_QUERY = "SELECT uri, _data, current_bytes, total_bytes, notificationextras FROM Downloads WHERE batch_id = ?";
```

Comment:
`current` and `total` bytes is not required for partial migrations. We want the `download-manager` to re-download these and get the size from the network call it will do!